app/pb_kits/playbook/pb_date/date.rb

```diff
@@ -33,7 +34,7 @@ def sm_date
     private
 
       def pb_date_time
-        Playbook::PbKit::PbDateTime.new(date)
+        Playbook::PbKit::PbDateTime.new(date, timezone)
```

Comment:
@coleerikson I wonder if someone would have passed timezone with date previously and this would cause it to pass `PbDateTime.new(date + timezone, timezone)`. Do you think this would cause a problem with the kit?

Reply:
**I think** if they passed the TZ, it's being ignored.

It's leveraging this: https://github.com/powerhome/playbook/blob/6.6.0/app/pb_kits/playbook/pb_kit/pb_date_time.rb#L8

Which has a second parameter, but as you can see it's not using it, therefore it's defaulting to EST.